qiita_pet/handlers/study_handlers/listing_handlers.py

```diff
@@ -172,3 +191,64 @@ def get(self):
         users, links = yield Task(self._get_shared_for_study, study)
 
         self.write(dumps({'users': users, 'links': links}))
+
+
+class SearchStudiesAJAX(BaseHandler):
+    def _get_standard(self, user, callback):
+        callback(_build_study_info("standard", user))
+
+    def _get_shared(self, user, callback):
+        """builds list of tuples for studies that are shared with user"""
+        callback(_build_study_info("shared", user))
+
+    @authenticated
+    def get(self, ignore):
+        search_type = self.get_argument('type')
+        user = self.get_argument('user')
```

Comment:
Why are you passing the user as an argument? It doesn't look like is needed as...